framework/src/application/node/processor/processor.js

```diff
@@ -187,7 +187,11 @@ class Processor {
 	}
 
 	async create(values) {
-		this.logger.trace({ data: values }, 'Creating block');
+		const { previousBlock, ...restOFValues } = values;
+		this.logger.trace('Creating block', {
```

Comment:
the data should come first know? and then the message?

Reply:
somehow it doesn't log if I give it as the second argument, for that, I have to stringify which can be done when using it as the second argument